### ISSUE TITLE
Error on startup

### ISSUE THREAD

**Issue by KetchupLord77** [Monika-After-Story/MonikaModDev#7307]
this is the error i get (btw i renamed "monika after story" to "monika")

```
I'm sorry, but an uncaught exception occurred.

While running game code:

File "game/zz_music_selector.rpy", line 14, in script

File "game/zz_music_selector.rpy", line 16, in <module>

ImportError: No module named mp3

-- Full Traceback ------------------------------------------------------------

Full traceback:

File "D:\games\Monika\DDLC-1.1.1-pc\renpy\
bootstrap.py", line 295, in bootstrap

renpy.main.main()

File "D:\games\Monika\DDLC-1.1.1-pc\renpy\
main.py", line 419, in main

game.context().run(node)

File "game/zz_music_selector.rpy", line 14, in script

File "D:\games\Monika\DDLC-1.1.1-pc\renpy\
ast.py", line 814, in execute

renpy.python.py_exec_bytecode(self.code.bytecode, self.hide,
store=self.store)

File "D:\games\Monika\DDLC-1.1.1-pc\renpy\
python.py", line 1719, in py_exec_bytecode

exec bytecode in globals, locals

File "game/zz_music_selector.rpy", line 16, in <module>

ImportError: No module named mp3

Windows-8-6.2.9200

Ren'Py 6.99.12.4.2187

Monika After Story 0.12.1
```

**Comment by Wingdinggaster656** [comment]
https://github.com/Monika-After-Story/MonikaModDev/wiki/Installation-and-Readme

The first thought is that your MOD is not installed properly. See that link.

Also, your _/game_ folder content, please?

**Comment by KetchupLord77** [comment]
@Wingdinggaster656  Ive had It installed for a while and it worked so I dont think I did anything wrong when I installed it, Here's the game folder
![image](https://user-images.githubusercontent.com/73570374/121857120-99f89580-cd38-11eb-8e1c-7a0f35752cf6.png)

**Comment by ThePotatoGuy** [comment]
folders look fine so is hard to say what is giong on here. If you are still having this issue on the latest version, then make a new issue.
